test/javascripts/acceptance/chat-browse-test.js

```diff
@@ -77,6 +106,37 @@ acceptance("Discourse Chat - chat browsing", function (needs) {
       "Follow button is present"
     );
   });
+
+  test("Previewing a channel", async function (assert) {
+    await visit("/chat/browse");
+    await click(".chat-channel-unfollow");
+    await click(".chat-channel-preview");
+
+    assert.ok(
```

Comment:
I think we could get away with just this test here, we are actually kind duplicating tests here, we want to ensure the card is showing, we already have various tests to assert the card is acting as we want it to